src/indicators/algorithms/exponentialMovingAverage.js

```diff
@@ -0,0 +1,49 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.indicators.algorithms.exponentialMovingAverage = function() {
+
+        var days = 9,
+            value = fc.utilities.fn.identity;
+
+        function undefinedArrayOfLength(length) {
```

Comment:
I was thinking about that too - I'll add that as part of the undefinedInputAdapter that I am working on seperately, as this will no doubt need to make use of this adapted property.